pkg/controllers/provisioning/scheduling/nodeset.go

```diff
@@ -28,14 +39,29 @@ import (
 )
 
 type NodeSet struct {
-	client          client.Client
+	kubeClient      client.Client
 	daemonResources v1.ResourceList
 	nodes           []*Node
+	topologies      map[uint64]*Topology
+	// Anti-affinity works both ways (if a zone has a pod foo with anti-affinity to a pod bar, we can't schedule bar to
+	// that zone, even though bar has no anti affinity terms on it. For this to work, we need to separately track the
+	// topologies of pods with anti-affinity terms, so we can prevent scheduling the pods they have anti-affinity to
+	// in some cases.
+	existingAntiAffinities map[uint64]*Topology
```

Comment:
For naming parity with topologies 
```suggestion
	antiAffinities map[uint64]*Topology
```

Reply:
Renamed to `inverseAntiAffinityTopologies` which is still not great, but more accurate.  The antiAffinities are also stored in `topologies.